src/main/java/com/prgrms/amabnb/room/api/RoomApiController.java

```diff
@@ -0,0 +1,27 @@
+package com.prgrms.amabnb.room.api;
+
+import com.prgrms.amabnb.room.entity.dto.request.CreateRoomRequest;
+import com.prgrms.amabnb.room.service.CreateRoomService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/rooms")
+@RequiredArgsConstructor
+public class RoomApiController {
+
+    private final CreateRoomService createRoomService;
+
+    @PostMapping
+    public ResponseEntity createRoom(@Valid @RequestBody CreateRoomRequest createRoomRequest) {
+        createRoomService.createRoom(createRoomRequest);
+        return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
```suggestion
        return ResponseEntity.created(uri).build();
```
생성된 자원에 대한 Location을 넣어주면 좋을 것 같아요!

Reply:
넵!!!